src/rendering/SceneManager.cc

```diff
@@ -2243,7 +2260,6 @@ SceneManager::LoadAnimations(const sdf::Actor &_actor)
   common::MeshManager *meshManager = common::MeshManager::Instance();
   descriptor.mesh = meshManager->Load(descriptor.meshName);
   common::SkeletonPtr meshSkel = descriptor.mesh->MeshSkeleton();
-
```

Comment:
This change is not related with the PR, do you mind to remove it ?

Reply:
Done.